static/app/components/profiling/flamegraph/flamegraph.tsx

```diff
@@ -209,6 +212,14 @@ function Flamegraph(): ReactElement {
     );
   }, [organization.features, profileGroup.metadata.platform]);
 
+  const hasBatteryChart = useMemo(() => {
+    const platform = profileGroup.metadata.platform;
+    return (
+      (platform === 'cocoa' || platform === 'android' || platform === 'node') &&
```

Comment:
why is node here? and does android report the battery as well?

Reply:
Heeem, no. This only needs to be cocoa